Innovator/Client/scripts/kanban/js/script.js

```diff
@@ -235,11 +235,17 @@ function loadBoards() {
 
 		// get boardimage
 		var vaultref = item.getProperty("creatorpicture","undefined");
-		var fileUrl = "nomanager.png";
-		/*if ((vaultref !== "undefined") && (vaultRefTable[vaultref] == undefined)) {
-			fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
-			vaultRefTable[vaultref] = fileUrl;
-		}*/
+		var fileUrl = "../kanban/nomanager.png";
+		if ((vaultref !== "undefined") && (vaultRefTable[vaultref] == undefined)) {
+			try{fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
```

Comment:
Fix the formatting of the try-catch statement.

Reply:
fixed.